app/scripts/lib/metrics.js

```diff
@@ -92,7 +91,7 @@ define(function (require, exports, module) {
     this._window = options.window || window;
 
     this._lang = options.lang || 'unknown';
-    this._context = options.context || 'web';
```

Comment:
@vladikoff - Hmm... I'm going to change `content_server_context` to `web` in constants.js to avoid hosing DataDog metrics.